RefactoringPracticeI/src/main/java/org/assuresoft/utils/TestConstants.java

```diff
@@ -0,0 +1,351 @@
+package org.assuresoft.utils;
+
+public final class TestConstants {
+  private TestConstants() {
+  }
+
+  public static final String EXPECTED_PROGRAM_OUTPUT = """
```

Comment:
Clever approach, as you may have noticed, this is waaay strict and usually not ideal, but we gotta be extreme when learning the "why", nice

Reply:
Yes the approval library doesn't work fine in java so I find this hardcode solution :)